pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
 packages = [
     {include = "poetry"},
 ]
+include = [{ path = "tests", format = "sdist" }]
```

Comment:
Sorry for the nitpick, but can you please add the entry on a new line similar to packages please.

Reply:
No problem!  Fixed in 5f0fe08.